src/libPMacc/test/random/2DDistribution.cu

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright 2016 Alexander Grund
+ *
+ * This file is part of libPMacc.
+ *
+ * libPMacc is free software: you can redistribute it and/or modify
+ * it under the terms of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * libPMacc is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with libPMacc.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pmacc_types.hpp"
+#include "memory/buffers/HostDeviceBuffer.hpp"
+#include "random/RNGProvider.hpp"
+#include "random/distributions/Uniform.hpp"
+#include "random/methods/Xor.hpp"
+#include "random/methods/XorMin.hpp"
+#include "random/methods/MRG32k3a.hpp"
+#include "random/methods/MRG32k3aMin.hpp"
+#include "dimensions/DataSpace.hpp"
+#include <stdint.h>
+#include <iostream>
+#include <fstream>
+
+typedef PMacc::DataSpace<2> Space2D;
```

Comment:
can you pls use our `DIM2` and `DIM3` defines here, since you are talking about *spatial* dimensions?